src/celeritas/decay/interactor/MuDecayInteractor.hh

```diff
@@ -0,0 +1,249 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/decay/interactor/MuDecayInteractor.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Constants.hh"
+#include "corecel/Macros.hh"
+#include "corecel/Types.hh"
+#include "corecel/data/StackAllocator.hh"
+#include "corecel/math/Algorithms.hh"
+#include "corecel/math/ArrayUtils.hh"
+#include "corecel/math/EulerRotation.hh"
+#include "celeritas/Quantities.hh"
+#include "celeritas/decay/data/MuDecayData.hh"
+#include "celeritas/phys/FourVector.hh"
+#include "celeritas/phys/Interaction.hh"
+#include "celeritas/phys/InteractionUtils.hh"
+#include "celeritas/phys/ParticleTrackView.hh"
+#include "celeritas/phys/Secondary.hh"
+#include "celeritas/random/distribution/BernoulliDistribution.hh"
+#include "celeritas/random/distribution/UniformRealDistribution.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Perform muon decay.
+ *
+ * Only one decay channel is implemented, with muons decaying to
+ * \f[
+ * \mu^- \longrightarrow e^- \overline{\nu}_e \nu_\mu
+ * \f]
+ * or
+ * \f[
+ * \mu^+ \longrightarrow e^+ \nu_e \overline{\nu}_\mu
```

Comment:
I actually did not really think when writing this, but the `\overline` and `\bar` are more of a stylistic preference. I see people using both, but for whatever reason `overline` usually renders better outside latex documents, so I just got used to it: $$\overline{\nu}$$ (overline) vs. $$\bar{\nu}$$ (bar). If we want to standardize and use `\bar`, I am okay with it.

Reply:
Overline is usually for multi-character bits, and bar is used for single character modifications right?

Reply:
Correct, I think you cannot do `\bar{xyz}`. So, always use `\bar` for anti-particles then? I reads better on the code (since many people actually say pbar instead of anti-proton), and makes more sense, and we accept the less nice rendering.